src/transformers/modeling_tf_utils.py

```diff
@@ -106,6 +106,19 @@ def get_config(self):
     return cls
 
 
+class TFCausalLanguageModelingLoss:
+    def compute_loss(self, labels, logits):
+        loss_fn = tf.keras.losses.SparseCategoricalCrossentropy(
+            from_logits=True, reduction=tf.keras.losses.Reduction.NONE
+        )
+        # make sure only labels that are not equal to -100
+        # are taken into account as loss
+        active_loss = tf.reshape(labels, (-1,)) != -100
```

Comment:
@jplu @sgugger @LysandreJik - In PyTorch we use -100 and not -1 to mask tokens for the loss. Should we do the same here? 
Would slightly break backward compatibility since -1 was already used for token classification - but not sure how many people already trained on token classification.

Reply:
I think -100 would be the most rigorous, right?

Reply:
Maybe we can release 3.0.1 immediately so that nearly no users are affected

Reply:
would be nice to have a consistent value there. Before it was only used for TokenClassification and we don't have any notebooks/ examples on TF token classification training so not too many people should be affected. I think it's worth it to align the values here

Reply:
+1 for consistency

Reply:
Ok to replace for consistency! but don't forget to update `run_tf_ner.py` and `TFTokenClassificationLoss` accordingly as well.

Reply:
Actually it might be better to deprecate `-1` here and still allow its usage for backward compatibility no? @sgugger @LysandreJik @jplu

Reply:
Yep! Good idea.